linode/manager
# upcoming: [M3-7855] - Updates to PATs for Restricted Parent Users

## Description 📝
As a Restricted Parent User, I should not be able to create Personal Access Tokens (PATs) with the 'child_account_access' grant if I do not possess the 'child_account_access' permission. This ensures that only users with the appropriate permissions can generate PATs with the 'child_account_access' grant. 
 
When viewing scopes, if the child_account_access grant for the parent user is later revoked, we'll still display this scope for existing PATs, but not for new ones.

## Target release date 🗓️
3/18

## Preview 📷
| Video |
| --- |
| <video src="https://github.com/linode/manager/assets/125309814/4d783e12-341c-4d92-9171-d63090ca8d49"/> |

## How to test 🧪

### Prerequisites
- Log into a unrestricted parent user account
- Log into a restricted parent user account (incognito)

### Reproduction steps
- Currently, when viewing scopes the `child account access` grant always appears
- Currently, when viewing the scope action menu in mobile view, a help icon/tooltip always appears when it should only appear for proxy users

### Verification steps
- Follow steps in video preview
- Make sure to check the `PUT /token` request payload in addition to what the drawer is/isn't displaying. 
- Then turn the feature flag *off* and confirm that when creating a PAT with all scopes set to Read Only, the `child_account` scope is not included in `scopes` in the payload. 
- Change screen width to mobile view and, on any non-proxy account, confirm that the help icon/tooltip is not visible

## As an Author I have considered 🤔

*Check all that apply*
